ouroboros-consensus/test-storage/Test/Ouroboros/Storage/VolatileDB/TestBlock.hs

```diff
@@ -154,9 +155,18 @@ createFileImpl :: IOLike m
                -> Internal.VolatileDBEnv m blockId
                -> m ()
 createFileImpl hasFS env = do
-    SomePair _stHasFS st <- Internal.getInternalState env
+    SomePair _stHasFS st <- getInternalState env
     let nextFd = Internal._nextNewFileId st
     let path = Internal.filePath nextFd
     withFile hasFS path (AppendMode MustBeNew) $ \_hndl -> do
         return ()
     return ()
+
+getInternalState :: forall m blockId. IOLike m
```

Comment:
I was more hoping that we wouldn't need access to the internal state of the VolatileDB in the tests anymore.